.github/workflows/workflows.json

```diff
@@ -17,6 +17,7 @@
   "appengine/typescript",
   "appengine/websockets",
   "appengine/twilio",
+  "cloudbuild",
```

Comment:
**Nitpick (feel free to ignore):**
Should the folder name be `cloud-build` to be consistent with existing `cloud-*` folders in this repo?

Reply:
I'm aligning with the the product API name and region tag, and think it's a minor bug where other directories don't but not necessarily worth fixing.